package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pillarwallet/pillarwallet-nodejs-sdk",
-  "version": "1.22.1",
+  "version": "1.22.2",
```

Comment:
As you're adding the new functionality it would be better to bump the middle number (22 => 23)